templates/ImageBuilder/cis-for-eb-image-pipeline-template.yaml

```diff
@@ -111,18 +111,43 @@ Resources:
                 inputs:
                   commands:
                     - sudo /usr/sbin/useradd --user-group xray -s /sbin/nologin --no-create-home
-              - name: CIS-5114
+              - name: CIS-5_1_14
                 # Edit ... /etc/rsyslog.conf ... and set $FileCreateMode to 0640 or more restrictive: $FileCreateMode 0640
                 action: ExecuteBash
                 inputs:
                   commands:
                     - sudo echo "\\$FileCreateMode 0640" >> /etc/rsyslog.conf
-              - name: CIS-5113
+              - name: CIS-5_1_13
                 # Ensure journald is configured to send logs to rsyslog. Edit the /etc/systemd/journald.conf file and add the following line: ForwardToSyslog=yes
                 action: ExecuteBash
                 inputs:
                   commands:
                     - sudo sed -i "s|#ForwardToSyslog=no|ForwardToSyslog=yes|g" /etc/systemd/journald.conf
+              - name: CIS-4_2_20
+                # Ensure SSH Idle Timeout Interval is configured
+                action: ExecuteBash
+                inputs:
+                  commands:
+                    - sudo sed -i "s|ClientAliveInterval 0|ClientAliveInterval 300|g" /etc/ssh/sshd_config
```

Comment:
This works as-is, but since the left-hand side is a regular expression we also have the ability future-proof against changes to default values with a wildcard match, e.g.:

`sed -i "s|ClientAliveInterval .*|ClientAliveInterval 300|" /etc/ssh/sshd_config`

Reply:
ah, I see, so in this particular case it doesn't matter.